lib/media/ttml_text_parser.js

```diff
@@ -331,6 +326,20 @@ shaka.media.TtmlTextParser.addStyle_ = function(
       }
     }
   }
+
+  var align = TtmlTextParser.getStyleAttribute_(
+      cueElement, region, styles, 'tts:textAlign');
+  if (align) {
+    if (align == 'center') {
+      // Need to set cue.position to auto to get centered
+      // subtitles
+      cue.align = 'middle';
+      cue.position = 'auto';
```

Comment:
Done! (And fixed the tests) 
Links to the bugs for references:
http://crbug.com/663797
https://bugzilla.mozilla.org/1316134